### PR TITLE
[BUG] Retain New Line Characters for String Value in Job Edit Form

### LINKED ISSUE #80: [BUG] Edit Jobs/Dirmon Entries is removing line breaks in text fields.
1. We want to retain the "\n" in text field for RJ to retain the header fields.
2. When editing a form, the \n characters are stripped from the field and when saved is removing header separations.

### PR DESCRIPTION
**Pull request by mjcloutier**
1. When editing a job the text fields are stripping the \n line breaks due to the field being sanitized in the view and thus losing the value on saving the form. We currently want to retain the line breaks.
2. Set minimum Ruby Version to 2.5
3. Convert \r\n line breaks to \n
4. Remove height for slice text_area fields.
5. Fix Batch Job Seed for proper slice numbering.

Closes #80

### CODE REVIEW

File: app/controllers/rocket_job_mission_control/jobs_controller.rb [hunk 1]
```diff
@@ -182,6 +182,9 @@ def update_slice
       # Finds specific slice [Array]
       slice = @job.input.failed.skip(offset).first
 
+      # Converts \r\n line breaks to \n
+      updated_records[0] = updated_records[0].gsub(/\r\n/, "\n")
```
Comment (reidmorrison) [inline app/controllers/rocket_job_mission_control/jobs_controller.rb:186]: Can we replace it with an iterator to change all records?
```suggestion
      updated_records.each { |record| record.gsub!(/\r\n/, "\n") }
```

File: app/models/rocket_job_mission_control/dirmon_sanitizer.rb [hunk 2]
```diff
@@ -42,8 +42,8 @@ def self.diff_properties(sanitized_properties, dirmon_entry)
             categories << props unless props.empty?
           end
           properties[:output_categories] = categories unless categories.empty?
-        else
-          properties[name] = value unless default_job.public_send(name) == updated_job.public_send(name)
+        elsif default_job.public_send(name) != updated_job.public_send(name)
+          properties[name] = value.is_a?(String) ? value.gsub(/\r\n/, "\n") : value
```
Comment (reidmorrison) [inline app/models/rocket_job_mission_control/dirmon_sanitizer.rb:46]:
```suggestion
          properties[name] = value.is_a?(String) ? value.gsub!(/\r\n/, "\n") : value
```

File: app/controllers/rocket_job_mission_control/jobs_controller.rb (hunk 1 above)
Comment (mjcloutier) [reply app/controllers/rocket_job_mission_control/jobs_controller.rb]: Added Correction.

File: app/models/rocket_job_mission_control/dirmon_sanitizer.rb (hunk 2 above)
Comment (mjcloutier) [reply app/models/rocket_job_mission_control/dirmon_sanitizer.rb]: Added Correction.

**Pull request merged**